deploy/cache-writer.yaml

```diff
@@ -18,7 +18,166 @@ kind: Template
 metadata:
   name: ccx-cache-writer
 objects:
-# TODO: Add the cache-writer deployment + services
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: valkey-writer
+    annotations:
+      ignore-check.kube-linter.io/minimum-three-replicas: "This app doesn't have that much traffic"
+  spec:
+    inMemoryDb: true
+    envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ccx
+    deployments:
+      - name: db-writer
+        minReplicas: ${{CACHE_WRITER_REPLICAS}}
+        webServices:
+          public:
+            enabled: false
+          private:
+            enabled: false
+          metrics:
+            enabled: true
+        podSpec:
+          env:
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__ADDRESSES
+              value: "${KAFKA_BOOTSTRAP_HOST}:${KAFKA_BOOTSTRAP_PORT}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__TIMEOUT
+              value: "${KAFKA_TIMEOUT}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__TOPIC
+              value: "${INCOMING_TOPIC}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__DEAD_LETTER_QUEUE_TOPIC
+              value: ""
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__PAYLOAD_TRACKER_TOPIC
+              value: "${PAYLOAD_TRACKER_TOPIC}"
+            - name: INSIGHTS_RESULTS_AGGREGATOR__BROKER__SERVICE_NAME
```

Comment:
is it safe to reuse the same service name as cache-writer? TBH I don't know where it is used. I thought we only use the group ID

Reply:
I changed it, just to be safe.